src/com/sindhu/webapplication/Timer.java

```diff
@@ -0,0 +1,26 @@
+package com.sindhu.webapplication;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class Timer {
+    public static void setTimer(String timerValue, int id) {
+        try {
+            String update = "UPDATE To_Do_Lists  SET timer='" + timerValue + "' WHERE ID='" + id + "'";
```

Comment:
I just wrote the sql query as string to update and used PreparedStatement for variable "update"